source/seismology/seismic-source.md

```diff
@@ -70,7 +70,13 @@ GCMT 地震目录中 1976-2005 年间全球地震的震级与频次关系。图
 引自《Introduction to Seismology》图 9.27。
 :::
 
-震级增加一级，地震释放的能量增加 32 倍。
+地震释放的能量与地震震级之间的关系为{cite}`Gutenberg1956`：
+
+$$
+\log E = 1.5 M + 11.8
+$$
+
+可以看出，震级增加一级，地震释放的能量增加 32 倍。
```

Comment:
When M is increased by 1, I can not get a 32*E from this equation. I cannot find the equation in the paper either.

Reply:
$\log E_1 = 1.5 M + 11.8$
$\log E_2 = 1.5 (M + 1) + 11.8$

=> $\log E_2/E_1 = 1.5$
=> $E_2/E_1 = 10^{1.5} = 31.6$

Reply:
I also can't find the exact equation in this paper, but many papers cited the Gutenberg paper. For example, Kanamori, 1977, JGR cited it:
<img width="523" alt="image" src="https://user-images.githubusercontent.com/3974108/202367922-82cc6d2b-0cb0-4652-88c9-9ea2e06ad8d1.png">